map_server/src/main.cpp

```diff
@@ -210,25 +210,25 @@ class MapServer
 #endif
       try {
         doc["resolution"] >> res;
-      } catch (YAML::InvalidScalar &) {
+      } catch (YAML::BadConversion &) {
```

Comment:
I don't know YAML exceptions well - but should we instead be using the parent class of both InvalidScalar and BadConversion - YAML::RepresentationException?